### PR TITLE
:white_check_mark: Adding tests for the limit of the preference field…

### PR DESCRIPTION
**Pull request by lagom1985**
… and the return of bad request.

### CODE REVIEW

File: src/test/java/com/codurance/allaboard/acceptance/UserEngagementSurveyFeature.java [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
```
Comment (isaporiti) [inline src/test/java/com/codurance/allaboard/acceptance/UserEngagementSurveyFeature.java]: Please remove this unused import.

**Review by isaporiti — CHANGES_REQUESTED** [review]

File: src/test/java/com/codurance/allaboard/acceptance/UserEngagementSurveyFeature.java (hunk 1 above)
Comment (lagom1985) [reply src/test/java/com/codurance/allaboard/acceptance/UserEngagementSurveyFeature.java]: done thanks

**Comment by isaporiti** [comment]
:shipit:

**Review by isaporiti — APPROVED** [review]
:shipit:

**Pull request merged**